src/Locale/en.json

```diff
@@ -7,6 +7,7 @@
   "APPETITE__SATISFACTORY": "Satisfactory",
   "AUTOMATED": "Automated",
   "BED_WITH_OXYGEN_SUPPORT": "Bed with Oxygen Support",
+  "BETA": "beta",
```

Comment:
use small case for all I18n strings. (Only make changes to the onces that you have introduced.)

Reply:
Sure making the changes.